src/util/camelCaseToUnderscore.spec.ts

```diff
@@ -0,0 +1,15 @@
+import camelCaseToUnderscore from "./camelCaseToUnderscore";
+
+describe("camelCaseToUnderscore", (): void => {
+  it("should return a string", (): void => {
+    expect(typeof camelCaseToUnderscore("fooBar")).toBe("string");
+  });
+
+  it("should convert a camelCase string to an underscore case", (): void => {
```

Comment:
I think this is a great opportunity for jest's `.each` functionality, template literal version:
https://jestjs.io/docs/en/api#2-testeach-table-name-fn-timeout
I'd also like to see many more tests, like the condition you have included that strips the beginning underscore, words with numbers in them, etc.

Reply:
Nice. I didn't know about `.each` very nice pattern for table driven tests. :+1: